.dependabot/config.yml

```diff
@@ -12,4 +12,5 @@ update_configs:
           dependency_name: '@types/node'
           version_requirement: '>= 9'
     version_requirement_updates: increase_versions
-    default_labels: []
+    default_labels: 
+      - "management/repo"
```

Comment:
why not just "dependabot" so it will be clear what this is about. Otherwise this knowledge hides in this obscure file.

Reply:
The point of this was to remove the otherwise pointless "dependencies" label. If you think it is preferable to have a unique label though, I can just leave this as is

Reply:
This is more of a cleaning issue rather than function.

Reply:
Why not `chore/dependencies`?

Reply:
@RomainMuller I think thats just semantics and leaves us with the same issue. If we really must have a label (which seems to be the case), and it should be a pointed one, I think having it be "dependencies" is fine.